EIPS/eip-1108.md

```diff
@@ -29,20 +33,30 @@ note](https://github.com/ethereum/go-ethereum/pull/16301#issuecomment-372687543)
 the computational cost of `ECADD`, `ECMUL`, and pairing checks (excepting the
 constant) has dropped roughly an order of magnitude across the board.
 
+Also, [optimisations in the bn library](https://github.com/paritytech/bn/pull/9) 
+used by the [Parity client](https://github.com/paritytech/parity-ethereum) led to a 
+significant performance boost we 
+[benchmarked](https://gist.github.com/pdyraga/4649b74436940a01e8221d85e80bfeef) 
+and compared against the [previous 
+results](https://github.com/ethereum/benchmarking/blob/master/constantinople/analysis2.md). 
+
 ## Specification
 
 Following is a table with the current gas cost and new gas cost:
 
 | Contract      | Address   | Current Gas Cost               | Updated Gas Cost    |
 | ------------- | --------- | -----------------------------  | ------------------- |
-| `ECADD`       | `0x06`    | 500<sup>[1]</sup>              | 50                  |
-| `ECMUL`       | `0x07`    | 40 000<sup>[1]</sup>           | 2 000               |
-| Pairing check | `0x08`    | 80 000k + 100 000<sup>[2]</sup>| 5 500k + 80 000     |
+| `ECADD`       | `0x06`    | 500<sup>[1]</sup>              | 150                 |
+| `ECMUL`       | `0x07`    | 40 000<sup>[1]</sup>           | 6 000               |
+| Pairing check | `0x08`    | 80 000k + 100 000<sup>[2]</sup>| 28 300k + 35 450    |
```

Comment:
Is this really 28300k = 28300000 gas ?

Reply:
This is a bit misleading but if you look at the EIP referenced in `[2]`, the [cost is expressed](https://github.com/ethereum/EIPs/blob/df132cd37efb3986f9cd3ef4922b15a767d2c54a/EIPS/eip-197.md#gas-costs) as:

The gas costs of the precompiled contract are `80 000 * k + 100 000`, where `k` is the number of points or, equivalently, the length of the input divided by 192.

The `k` seems like a wrong choice of a letter but we did not want to move away from that notation to do not cause even more confusion.

Reply:
To be clear the cost is expressed as `80 000 * k + 100 000`. I'd suggest you use the same notation to avoid confusion, that is, do not omit the multiplication sign :)

Reply:
Oh, yeah! My fault. Will fix it in a second...